internal/project/project.go

```diff
@@ -460,6 +484,8 @@ func (p *Project) updateGraph() bool {
 		return false
 	}
 
+	defer p.host.fs.ClearCache()
```

Comment:
But this still keeps cache outside of upate graph right. You probably want cache to be present only during updateGraph and use uncached version outside of it, that is it should not cache not consult cache when outside the updateGraph call

Reply:
The trouble is that the entire project is "a host", so it's pretty tough to do that, unless I make the cached FS able to be disabled entirely on some sort of toggle, which seems odd...

(This is why I hate every object turning into a host, but here we are...)

Reply:
Maybe what I should do is create the cache in Program, which is how it's done for the other code (`NewCachedFSCompilerHost`).

Reply:
Well, I did it, but it's not pretty 😄

Reply:
I think you want to set  and unset cacheHost inself ? so that its not checked all over the place?

also i think ata needs to used uncached version all the time? since it could be doing things in background while project is updating